### DISCUSSION TITLE
Не получается включить микрофон в звонке и в голосовых чатах в каналах

### DISCUSSION THREAD

**Discussion by dedhaggis** [stoatchat/discussions#819] [Q&A]
В настройках при установке микрофона и наушников в настройках голоса при перезаходе в настройки голоса все сбрасываеться

**Comment by thewoolfi** [comment]
Это визуальный баг, а так все работает. Присоединяешься, даёшь разрешение и говоришь.

WOAH (RU)

**Reply by qcklll** [reply]
@thewoolfi Привет, а подскажи, пожалуйста, разрешение нужно давать каждый раз? 
Пробовал и через настройки винды в том числе

**Reply by Mellin-A** [reply]
Разрешение можно не давать вовсе. Это настройка для изминения активного оборудования.

**Reply by VasiliyKos** [reply]
у меня ни в 1 чате голосовом не работают микрофон и наушники, помечены красным, как-будто отключены, и также у друзей. как исправить?
![Screenshot_2](https://github.com/user-attachments/assets/996e5a97-fd91-4b7f-84f1-7e5828ecb23b)

**Reply by Mellin-A** [reply]
> у меня ни в 1 чате голосовом не работают микрофон и наушники, помечены красным, как-будто отключены, и также у друзей. как исправить? ![Screenshot_2](https://private-user-images.githubusercontent.com/116308238/375940200-996e5a97-fd91-4b7f-84f1-7e5828ecb23b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MjI5MzIsIm5iZiI6MTcyODcyMjYzMiwicGF0aCI6Ii8xMTYzMDgyMzgvMzc1OTQwMjAwLTk5NmU1YTk3LWZkOTEtNGI3Zi04NGYxLTdlNTgyOGVjYjIzYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMlQwODQzNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYmU1ODQ0NWVjYzFmMTE1NDIzYTkwNTU5NGRiNzBiNzFlYWI4MzZhNmE3ODliOTNjNTUwMDNhNzQwNGFkZjUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6NaI6vc46Nxs1y2WHXLKQ1BiwSx2Bf6VKGfIO76Vnoo)

Перезаходи до тех пор, пока возле зеленых столбиков не появится информация об успешном подключении
![image](https://github.com/user-attachments/assets/23b3cc61-851e-411b-903e-b1d4de64b64e)

**Reply by RAgon23** [reply]
> 
> Перезаходи до тех пор, пока возле зеленых столбиков не появится информация об успешном подключении !

ну фиг знает, более 50 переподключений и в приложении и в браузере, результата ноль. Либо серваки умирают от нагрузки, либо подключение к войсу специально было седлано максимально неудобно, чтобы нерабочей фигней никто не пользовался

**Reply by WolfySoCute** [reply]
> > Перезаходи до тех пор, пока возле зеленых столбиков не появится информация об успешном подключении !
> 
> ну фиг знает, более 50 переподключений и в приложении и в браузере, результата ноль. Либо серваки умирают от нагрузки, либо подключение к войсу специально было седлано максимально неудобно, чтобы нерабочей фигней никто не пользовался

Мне хватало и 3-10 переподключений. В настройках же написано, что войсы переписывают, вот они и не работают
